modules/_nixos-disintegration/kernel.nix

```diff
@@ -14,13 +14,13 @@ let
 in
 {
   disabledModules = [
-    <nixpkgs/nixos/modules/system/boot/kernel.nix>
+    "system/boot/kernel.nix"
   ];
 
   imports = [
     (
       let
-        toOverride = import <nixpkgs/nixos/modules/system/boot/kernel.nix> {
+        toOverride = import "${_mobile-nixos.nixpkgsPath}/nixos/modules/system/boot/kernel.nix" {
```

Comment:
Gnarly, but it's required due to working around `kerelPackage`'s `.apply` function. Though AFAIUI this should still be hermetic with regards to the used Nixpkgs.